### DISCUSSION TITLE
Tutorial execute file couldn't found in directory

### DISCUSSION THREAD

**Discussion by seongwonHam** [idaholab/moose#21747] [Q&A Getting Started] [answered]
Hello, MOOSE users.

I'm starter of MOOSE and I ran some examples to understand MOOSE.
Today, I start to tutorial02_multiapps in /projects/moose/tutorials/tutorial02_multiapps, but I can't found execute file in this path, there only exist some input files.

I have some experience of MOOSE example and tutorial01, but I had ran the program with the input file and -opt file to calculate it. So, I wonder how can I execute the tutorial02 and get some output file at /projects/moose/tutorials/tutorial02_multiapps.

Please let me know how can I do this.

Thanks.

**Comment by GiudGiud** [comment, marked as answer]
Hello

The app is in `moose/tutorials/tutorial02_multiapps/app` and you may build the executable in that folder. Then you will have to refer to it to run your input files. 
You could also probably build just any MOOSE executable for this tutorial, they would all work for this tutorial. 

Guillaume

**Reply by seongwonHam** [reply]
Oh, I see. I'll follow it.
Thanks!
